lib/pages/athlete/AthletePage.dart

```diff
@@ -987,9 +987,10 @@ class _AthletePageState extends State<AthletePage> {
                       ),
                     ],
                   ),
-                  SizedBox(width: 30),
```

Comment:
I do think we should add a bit of spacing here other wise the add to wallet button will look too close to the seasonal APT title and the long/short toggle